source/MaterialXRender/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ if(UNIX)
 endif()
 
 if(MATERIALX_BUILD_OIIO)
-    list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/External/OpenImageIO")
     find_package(OpenImageIO CONFIG REQUIRED)
     target_link_libraries(${TARGET_NAME} PRIVATE OpenImageIO::OpenImageIO OpenImageIO::OpenImageIO_Util)
+    target_compile_definitions(MaterialXGraphEditor PUBLIC MATERIALX_BUILD_OIIO)
```

Comment:
Makes sure that any project linking with MaterialXRender, both locally and in external projects, are aware that this MaterialX was built with OIIO support. The info will be written in the generated CMake for this library.

Please note that the `target_link_library` can stay `PRIVATE` since `OiioImageLoader.h` hides the underlying OIIO implementation. It still needs the compile definition to make the class visible.